common/lib/xmodule/xmodule/partitions/partitions.py

```diff
@@ -2,12 +2,23 @@
 
 from collections import namedtuple
 from stevedore.extension import ExtensionManager
+from xblock.fields import List
 
 # We use ``id`` in this file as the IDs of our Groups and UserPartitions,
 # which Pylint disapproves of.
 # pylint: disable=redefined-builtin
 
 
+class UserPartitionList(List):
+    """Special List class for listing UserPartitions"""
```

Comment:
Also, if either of you think that docstrings aren't worth linting on, drop a note here: https://github.com/edx/edx-lint/pull/53.